dietpi/dietpi-software

```diff
@@ -6800,28 +6801,40 @@ _EOF_
 			Download_Test_Media
 		fi
 
-		software_id=68 # Remot3.it
+		software_id=68 # Remote.It
 		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
 
 			Banner_Installing
 
+			local licensekey=
+            G_WHIP_BUTTON_CANCEL_TEXT='Skip'
+			G_WHIP_INPUTBOX 'If you do not already have the Remote.It Desktop application on your laptop, install it.
+https://www.remote.it/download\n
+Please enter your license key obtained from the Remote.It Desktop application.
+[Account] -> [Licensing]\n\n
+NB: This is optional, select "Skip" to leave it empty.
+    If so, register the device using the claim code shown during the installation process.\n
+    See below for instructions on how to register the device using a claim code:
+    https://docs.remote.it/software/device-package/installation#3.-claim-and-register-the-device\n' && licensekey=$G_WHIP_RETURNED_VALUE
+            G_EXEC export R3_REGISTRATION_CODE="$licensekey"
+            G_EXEC export R3_PLATFORM_CODE="1072"
+
 			# ARMv6/7
-			local arch='armhf'
+			local url='https://downloads.remote.it/remoteit/latest/remoteit.armhf.deb'
 
 			# ARMv8
 			if (( $G_HW_ARCH == 3 )); then
 
-				arch='arm64'
+				local url='https://downloads.remote.it/remoteit/latest/remoteit.arm64.deb'
 
 			# x86_64
 			elif (( $G_HW_ARCH == 10 )); then
 
-				arch='amd64'
+				local url='https://downloads.remote.it/remoteit/latest/remoteit.amd64.deb'
 
 			fi
 
-			local fallback_url="https://github.com/remoteit/installer/releases/download/v2.6.39/connectd_2.6.39_$arch.deb"
-			Download_Install "$(curl -sSfL 'https://api.github.com/repos/remoteit/installer/releases/latest' | mawk -F\" "/\"browser_download_url\": .*\/connectd_[^\"\/]*_$arch\.deb\"/{print \$4}")"
+			Download_Install "$url"
 
```

Comment:
To implement the suggestion [below](https://github.com/MichaIng/DietPi/pull/5790/files#r990152764):
```suggestion

if dpkg-query -s 'connectd' &> /dev/null || dpkg-query -s 'weavedconnectd' &> /dev/null
then
		G_WHIP_BUTTON_CANCEL_TEXT='Skip' G_WHIP_YESNO '[WARNING] Legacy Remote.It package detected
\nOne of the legacy Remote.It packages "connectd" or "weavedconnectd" is installed on your system. These have been superseded by the new "remoteit" package.
\nDo you want the legacy packages to be uninstalled now?
\nNB: You can do this any time later, e.g. after connections with the new package have been set up:
    G_AGP connectd weavedconnectd' && G_AGP connectd weavedconnectd
fi
```

Reply:
Thanks. The code has been modified.